### ISSUE TITLE
[HIPIFY] hipify-clang goes to install_root/ instead of  install_root/bin

### ISSUE THREAD

**Issue by Kaveh01** [ROCm-Developer-Tools/HIP#1571]
After commit f19e7c29dfcef3d0c941c977ce6d1448ad4e288a hipify-clang goes to the install_root/
and not to the install_root/bin:

`rm -rf ~/software/hip && rm -rf * && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_HIPIFY_CLANG=ON -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DHIP_COMPILER=clang   -DCMAKE_INSTALL_PREFIX=/home/kaveh/software/hip ../ && make -j8`

```
$ ls ~/software/hip 
bin  cmake  hipify-clang  include  src
$ ls ~/software/hip/bin/ 
findcode.sh                hipconfig                  hipexamine-perl.sh
finduncodep.sh             hipconvertinplace-perl.sh  hipexamine.sh
hipcc                      hipconvertinplace.sh       hipify-cmakefile
hipcc_cmake_linker_helper  hipdemangleatp             hipify-perl
```

**Comment by emankov** [comment]
Please verify.

**Comment by Kaveh01** [comment]
When I am not specifying "-DBIN_INSTALL_DIR" , it still installs hipify-clang in the install_root:
 
```
$ rm -rf ~/software/hip && rm -rf * && cmake -DCMAKE_BUILD_TYPE=Release
 -DBUILD_HIPIFY_CLANG=ON -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DHIP_COMPILER=clang  
 -DCMAKE_INSTALL_PREFIX=/home/kaveh/software/hip ../ && make -j8 install
...
-- Installing: /home/kaveh/software/hip/include/avxintrin.h
-- Installing: /home/kaveh/software/hip/include/vadefs.h
-- Installing: /home/kaveh/software/hip/include/float.h
-- Installing: /home/kaveh/software/hip/include/avx512vp2intersectintrin.h

$ ls ~/software/hip
bin  cmake  hipify-clang  include  src
$ ls ~/software/hip/bin
findcode.sh                hipconfig                  hipexamine-perl.sh
finduncodep.sh             hipconvertinplace-perl.sh  hipexamine.sh
hipcc                      hipconvertinplace.sh       hipify-cmakefile
hipcc_cmake_linker_helper  hipdemangleatp             hipify-perl
```
```
and when I put "-DBIN_INSTALL_DIR" it installes everything in BIN_INSTALL_DIR:
rm -rf ~/software/hip && rm -rf * && cmake -DBIN_INSTALL_DIR=/home/kaveh/software/hip/bin 
-DCMAKE_BUILD_TYPE=Release -DBUILD_HIPIFY_CLANG=ON 
-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DHIP_COMPILER=clang
-DCMAKE_INSTALL_PREFIX=/home/kaveh/software/hip ../ && make -j8 install
...
-- Installing: /home/kaveh/software/hip/bin/include/bmiintrin.h
-- Installing: /home/kaveh/software/hip/bin/include/rdseedintrin.h
-- Installing: /home/kaveh/software/hip/bin/include/__wmmintrin_pclmul.h
-- Installing: /home/kaveh/software/hip/bin/include/avxintrin.h
-- Installing: /home/kaveh/software/hip/bin/include/vadefs.h
-- Installing: /home/kaveh/software/hip/bin/include/float.h
-- Installing: /home/kaveh/software/hip/bin/include/avx512vp2intersectintrin.h

$ ls ~/software/hip/
bin
$ ls ~/software/hip/bin/
bin  cmake  hipify-clang  include  src
kaveh@zam1283:~/source/HIP/build$ ls ~/software/hip/bin/bin/
findcode.sh                hipconfig                  hipexamine-perl.sh
finduncodep.sh             hipconvertinplace-perl.sh  hipexamine.sh
hipcc                      hipconvertinplace.sh       hipify-cmakefile
hipcc_cmake_linker_helper  hipdemangleatp             hipify-perl
```

**Comment by Kaveh01** [comment]
Fixed with 6986818172a985660d367109dc53f84728b560cf .
